main.py

```diff
@@ -13,7 +13,7 @@
 
 users = {}
 notes = {}
-secret_key = os.environ["PASTEBIN_JWT_SECRET"]
+secret_key = 'secret'#os.environ["PASTEBIN_JWT_SECRET"]
```

Comment:
Should 'secret' be hard-coded here?

Reply:
I assume we have this to see if the secret scanning works properly?

Reply:
yes, was testing this and trufflehog not detecting it.. ideally this would go in github secrets (codespaces)

Reply:
Do we want to revert it in this PR or leave it for further testing with trufflehog?

Reply:
i think we can revert it.. we can use a new variable with secret if we want to continue testing trufflelog

Reply:
Sounds good. Lets do that.

Reply:
As discussed in Slack, update this to something that Trufflehog will catch as an unverified secret.